### PR TITLE
Add command to open tab and focus search dialog

### PR DESCRIPTION
**Pull request by parantapa**
Added a simple command to open new tab and highlight the search dialog.

### CODE REVIEW

File: extension/lib/commands.coffee [hunk 1]
```diff
@@ -580,6 +585,7 @@ commands = [
   new Command('nav',    'scroll_page_up',        command_scroll_page_up,        [['<s-space>']])
 
   new Command('tabs',   'open_tab',              command_open_tab,              [['t']])
+  new Command('tabs',   'open_tab_serach',       command_open_tab_search,       [['T'], ['S']])
```
Comment (cesalazar) [inline extension/lib/commands.coffee]: Typo: 'open_tab_serach'

**Comment by lydell** [comment]
Try opening the help dialog with your patch :P 

I’m not sure how useful this command is. Personally, I never ever use the search bar—I search from the location bar.

Since this new command only is a combination of two existing commands, I think it might be better to wait until VimFx supports custom commands. However, that is very far into the future. Unless there is great interest in this feature, the best thing I can recommend is to run your own custom build of VimFx (that’s what I do: latest develop + PRs waiting to be merged into develop + my own custom commands).

**Comment by parantapa** [comment]
Using my custom build of VimFx is what I am doing right now.

However coming from Pentadactyl I have developed muscle memory to hit Shift-S and getting the open tab and search command. I just thought it might be useful for other as well.

**Comment by lydell** [comment]
Sorry, this won’t be merged. However, the latest development version supports custom commands. I’ve added [Open new tab and focus Search Bar](https://github.com/akhodakivskiy/VimFx/wiki/Custom-Commands#open-new-tab-and-focus-search-bar) to the wiki.

**Pull request closed without merging**